src/web_app/js/windowport.js

```diff
@@ -0,0 +1,49 @@
+/*
+ *  Copyright (c) 2015 The WebRTC project authors. All Rights Reserved.
+ *
+ *  Use of this source code is governed by a BSD-style license
+ *  that can be found in the LICENSE file in the root of the source
+ *  tree.
+ */
+
+/* More information about these options at jshint.com/docs/options */
+
+/* globals trace, chrome */
+/* exported WindowPort */
+
+'use strict';
+
+// This is used to communicate from the Chrome App window to background.js.
+// It opens a Port object to send and receive messages. When the Chrome
+// App window is closed, background.js receives notification and can
+// handle clean up tasks.
+var WindowPort = (function() {
```

Comment:
For namespace can we follow this pattern? (I think it's cleaner than repeating declaration at the end of the file. Also see style guide for namespace naming conventions).

var apprtc = apprtc || {};
apprtc.port = apprtc.port || {};
(function() {
  var port_ = chrome.runtime.connect();
  apprtc.port.sendMessage = function(message) {...}
  apprtc.port.addListener = function(listener) {...}
})();